user_templates/mcu.cfg

```diff
@@ -12,4 +12,3 @@
 # [board_pins] section. You can use the pinout documentation here: ./docs/pinout.md
 
 # ------------------------------------------------------------------------------------------
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates inside from the install script